src/components/app/SymbolicationStatusOverlay.js

```diff
@@ -61,11 +61,6 @@ class SymbolicationStatusOverlay extends PureComponent {
   }
 }
 
-SymbolicationStatusOverlay.propTypes = {
-  symbolicationStatus: PropTypes.string.isRequired,
-  waitingForLibs: PropTypes.object.isRequired,
-};
-
```

Comment:
I came to see this file while I was looking for symbolicationStatus; it's only user is using flow so we can remove the prop types typing, but tell me if you'd like this to go to a separate PR instead.